docs/fidesops/docs/postman/Fidesops.postman_collection.json

```diff
@@ -3427,7 +3427,12 @@
 						"header": [],
 						"body": {
 							"mode": "raw",
-							"raw": "{\n    \"meta\": {\"version\": \"0.5\"},\n    \"regime\": \"ccpa\",\n    \"exercise\": [\n        \"access\"\n    ],\n    \"identity\": \"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InRlc3RAZXhhbXBsZS5jb20ifQ.4I8XLWnTYp8oMHjN2ypP3Hpg45DIaGNAEmj1QCYONUI\",\n}"
+							"raw": "{\n    \"meta\": {\"version\": \"0.5\"},\n    \"regime\": \"ccpa\",\n    \"exercise\": [\n        \"access\"\n    ],\n    \"identity\": \"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InRlc3RAZXhhbXBsZS5jb20ifQ.4I8XLWnTYp8oMHjN2ypP3Hpg45DIaGNAEmj1QCYONUI\"\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
```

Comment:
sorry what was the error here, @eastandwestwind?

Reply:
previously was using raw `Text` instead of raw `JSON`. Also, the extra comma after `identity` was causing an error.

Reply:
got it thank you